features/steps/given.php

```diff
@@ -102,7 +102,7 @@ function ( $world, TableNode $table ) {
 	}
 );
 
-$steps->Given( '/^save (STDOUT|STDERR) ([\'].+[^\'])?as \{(\w+)\}$/',
+$steps->Given( '/^save (STDOUT|STDERR) ([\'].+[^\'])?\s?as \{(\w+)\}$/',
```

Comment:
This source of this code is WP-CLI's test suite. Want to submit a PR there too? https://github.com/wp-cli/wp-cli/blob/master/features/steps/given.php#L112

Related https://github.com/wp-cli/wp-cli/issues/3826